pkg/kn/commands/revision_get_test.go

```diff
@@ -28,7 +29,21 @@ import (
 	client_testing "k8s.io/client-go/testing"
 )
 
-func fakeRevisionGet(args []string, response *v1alpha1.RevisionList) (action client_testing.Action, output []string, err error) {
+func fakeRevisionGet(args []string, response interface{}, listOrsingle bool) (action client_testing.Action, output []string, err error) {
+	var (
+		resp runtime.Object
+		ok   bool
+	)
+	if listOrsingle {
```

Comment:
I don't get this conditional. if and else branch are the same, aren't they ?

Reply:
Yes they are same..
Its my fault i was thinking some other scenario to handle and wrote that.
I will change and update